CMake/resolve_dependency_modules/arrow/CMakeLists.txt

```diff
@@ -72,11 +79,15 @@ if(VELOX_ENABLE_ARROW)
     URL ${VELOX_ARROW_SOURCE_URL}
     URL_HASH ${VELOX_ARROW_BUILD_SHA256_CHECKSUM}
     SOURCE_SUBDIR cpp
+    CMAKE_COMMAND ${CMAKE_COMMAND}
+    -E
+    env
+    ARROW_THRIFT_URL=${THRIFT_URL}
```

Comment:
This might be confusing for people not so familiar with cmake, can we add a comment explaining that this is needed, because ep doens't inherit (env) variables?